tests/std/tests/P0896R4_stream_iterators/test.cpp

```diff
@@ -117,4 +247,19 @@ int main() {
 
     test_ostreambuf_iterator(cout);
     test_ostreambuf_iterator(wcout);
+
+    test_regex_iterator("hello world");
+    test_regex_iterator(L"hello world");
+    test_regex_iterator(string("hello world"));
+    test_regex_iterator(wstring(L"hello world"));

```

Comment:
No change requested (this isn't worth resetting testing): In new code, we generally prefer to construct temporaries with braces when possible, as this makes them visibly different from function calls. This is a relatively minor preference, especially in test code, and the codebase is far from consistent at this time.